src/vs/editor/contrib/inlineEdit/browser/inlineEditController.ts

```diff
@@ -233,7 +233,8 @@ export class InlineEditController extends Disposable {
 		}
 		const line = edit.range.endLineNumber;
 		const column = edit.range.endColumn;
-		const ghostText = new GhostText(line, [new GhostTextPart(column, edit.text, false)]);
+		const textToDisplay = edit.text.endsWith('\n') && !(edit.range.startLineNumber === edit.range.endLineNumber && edit.range.startColumn === edit.range.endColumn) ? edit.text.slice(0, -1) : edit.text;
```

Comment:
I think there is `isEmpty`:
```suggestion
		const textToDisplay = edit.text.endsWith('\n') && !(edit.range.isEmpty()) ? edit.text.slice(0, -1) : edit.text;
```

Also, for better readability you could use some `trimEnd(edit.text, '\n')` function.